Cargo.toml

```diff
@@ -16,3 +16,21 @@ reqwest = { version = "0.12.7", features = ["blocking"] }
 postgres = "0.19.9"
 trust-dns-client = "0.23.2"
 lettre = "0.11.9"
+
+[lints.rust]
```

Comment:
Are you sure it works on stable Rust? https://www.reddit.com/r/rust/comments/13re6rr/you_can_specify_lints_with_lints_table_in/

Reply:
yes, stable for a while -- https://doc.rust-lang.org/cargo/reference/manifest.html#the-lints-section